tests/integration/cartography/intel/aws/test_eks.py

```diff
@@ -0,0 +1,30 @@
+import cartography.intel.aws.eks
+import tests.data.aws.eks
+
+TEST_ACCOUNT_ID = '000000000000'
+TEST_REGION = 'eu-west-1'
+TEST_UPDATE_TAG = 123456789
+
+
+def test_load_eks_clusters(neo4j_session):
+    data = tests.data.aws.eks.DESCRIBE_CLUSTERS
+
+    cartography.intel.aws.eks.load_eks_clusters(
+        neo4j_session,
+        data,
+        TEST_REGION,
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+    expected_nodes = {
+        "arn:aws:eks:eu-west-1:111111111111:cluster/cluster_1",
+        "arn:aws:eks:eu-west-2:222222222222:cluster/cluster_2",
+    }
+
+    nodes = neo4j_session.run(
+        """
+        MATCH (r:EKSCluster) RETURN r.arn;
```

Comment:
I think we should add test to show connection between AWSAccount and the EKSCluster - verify that `(:AWSAccount)--(:EKSCluster)` exists.

Reply:
I've added the function `test_load_eks_clusters_relationships` below. Let me know if it's ok for you!